templates/archives/stories/whirlwind.jinja2

```diff
@@ -0,0 +1,66 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+The Rise and Fall of Project Whirlwind
+{% endblock %}
+
+{% block story_credit %}
+Meesue Kim
+{% endblock %}
+
+
+{% block story_body %}
+<p>
+    Arguably the first large-scale and high-speed computer, the Whirlwind was developed by the MIT Servomechanisms
+    Laboratory in a project directed by Jay Forrester for the United States Navy. When the project started in 1944, it
```

Comment:
Link Jay Forrester to People archive for him  : /archives/person/jay-w-forrester